tools/eslint-rules/timer-arguments.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @fileoverview Require at least two arguments when calling setTimeout() or
+ * setInterval().
+ * @author Rich Trott
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function isTimer(name) {
+  return ['setTimeout', 'setInterval'].includes(name);
+}
+
+module.exports = function(context) {
+  return {
+    'CallExpression': function(node) {
+      const name = node.callee.name;
+      if (isTimer(name) && node.arguments.length < 2) {
+        context.report(node, `${name} must have exactly 2 arguments`);
```

Comment:
This error message seems a bit misleading to me, since `setTimeout` and `setInterval` can be called with more than 2 arguments. Maybe it would be better to use `must have at least 2 arguments` instead.